Tests/BluetoothTests/GATTCharacteristicTests.swift

```diff
@@ -339,4 +341,63 @@ final class GATTCharacteristicTests: XCTestCase {
         XCTAssertEqual(GATTAge.uuid, .age)
         XCTAssertEqual(Year.unitType, .year)
     }
+    
+    func testAnaerobicHeartRateLowerLimit() {
+        
+        typealias BeatsPerMinute = GATTAnaerobicHeartRateLowerLimit.BeatsPerMinute
+        
+        XCTAssertNil(GATTAnaerobicHeartRateLowerLimit(data: Data([0x3d, 0x72])))
+        
+        let data = Data([0x22])
+        
+        let beats: BeatsPerMinute = 34
+        
+        guard let characteristic = GATTAnaerobicHeartRateLowerLimit(data: data)
+            else { XCTFail("Could not decode from bytes"); return }
+        
+        XCTAssertEqual(characteristic.data, data)
+        XCTAssertEqual(characteristic.beats, beats)
+        XCTAssertEqual(characteristic.description, "34")
+        XCTAssertEqual(beats.description, "34")
+        XCTAssertEqual(GATTAnaerobicHeartRateLowerLimit.uuid, .anaerobicHeartRateLowerLimit)
+        XCTAssertEqual(BeatsPerMinute.unitType, .beatsPerMinute)
+    }
```

Comment:
You are missing a Equatable test of GATTAnaerobicHeartRateLowerLimit

Reply:
Add `XCTAssertNotEqual()` as well